org.kde.WaylandDecoration.QWhiteSurGtkDecorations.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "org.kde.WaylandDecoration.QWhiteSurGtkDecorations",
+    "branch": "5.15-23.08",
+    "runtime": "org.kde.Platform",
+    "build-extension": true,
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.15-23.08",
+    "appstream-compose": false,
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "qwhitesurgtkdecorations",
+            "buildsystem": "cmake-ninja",
+            "config-opts": ["-DCMAKE_INSTALL_PREFIX=/usr/share/runtime/lib/plugins/QWhiteSurGtkDecorations",
+                            "-DQT_PLUGINS_DIR=/usr/share/runtime/lib/plugins/QWhiteSurGtkDecorations"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/FengZhongShaoNian/QWhiteSurGtkDecorations.git",
+                    "tag": "0.1.0"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
```

Comment:
this should be part of the main module.

Reply:
Sorry, I'm new to Flatpak and don't quite understand what you mean

Reply:
Add

```
                {
                    "type": "file",
                    "path": "org.kde.WaylandDecoration.QWhiteSurGtkDecorations.appdata.xml"
                }
```

to the source above, then add the install command in `post-install`.

Reply:
Ok, thanks!